src/utils/messages.json

```diff
@@ -54,5 +54,6 @@
   "internship.page.table.role": "Role",
   "internship.page.table.position": "Position",
   "internship.page.table.team": "Team",
-  "candidates.popper.inputPlaceholder": "Search..."
+  "candidates.popper.inputPlaceholder": "Search...",
+  "internships.popper.inputPlaceholder": "Search..."
```

Comment:
Why did you duplicate code ? Instead of candidates.popper.inputPlaceholder and internships.popper.inputPlaceholder use one common field

Reply:
didn’t know that you can use the same one, I’ll fix it